Tests/Directory.Build.targets

```diff
@@ -0,0 +1,2 @@
+<Project>
```

Comment:
Is this file required?

Reply:
It is required to avoid leaking the roots `Directory.Build.targets` into the tests project.